README.md

```diff
@@ -16,9 +16,9 @@ In my use case, tracking many results of functions on server and upload to S3 (I
 $ pip install decotra
 ```
 
-and if you use local s3 server (like a minio), please set a environmental parameters `S3_ENDPOINT_URL`.
```

Comment:
`like` is not formal expression. Sentences in formal documents do not start with `And` or `But`.